src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -64,4 +64,8 @@
         <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_LOG_DEBUG_VERSION" />
     </ItemGroup>
 
+    <!-- Apple Privacy Manifest file, more information: https://aka.ms/maui-privacy-manifest -->
+    <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
```

Comment:
Doesn't this apply to catalyst as well?

Reply:
No, as far as we know it's only required for iOS and tvOS.

Reply:
Thios should be automatically added to the build if it is in the `Platforms\iOS\Resources` folder. No need to edit the csproj.

Reply:
Moved it to `Platforms\iOS\Resources`, but seems like something is wrong with being picked up there. Raised internally.